### ISSUE TITLE
esp32-lyratd-syna音頻板支持

### ISSUE THREAD

**Issue by RickChou** [espressif/esp-va-sdk#108]
Hi espressif Team

我想買esp32-lyratd-syna dsp板。但想要改成配置synaptics audio board，我該如何做呢??

Best Regards.

**Comment by expresspotato** [comment]
Yes, I also want to test the Synaptics CX2921, yet there is no firmware for the chip included in the repo...
